include/dsAVDTypes.h

```diff
@@ -779,6 +779,13 @@ typedef enum _dsDisplayMatrixCoefficients_t
     dsDISPLAY_MATRIXCOEFFICIENT_MAX            ///< Out of range
 } dsDisplayMatrixCoefficients_t;
 
+typedef enum _dsAudioInput_t
```

Comment:
add @brief and @note Used in @link

Reply:
Added

Reply:
Please specify "MS12 only" as this mixer is used for MS12 only.

Reply:
Updated

Reply:
remove -> _dsAudioInput_t it's not required